src/abc_atlas_access/abc_atlas_cache/cloud_cache.py

```diff
@@ -432,6 +430,98 @@ def __init__(self,
         # last loaded from this cache dir (if applicable)
         self._manifest_last_used = c_path / '_manifest_last_used.txt'
 
+        # self._downloaded_data_path is where we will keep a JSONized
+        # dict mapping paths to downloaded files to their file_hashes;
+        # this will be used when determining if a downloaded file
+        # can instead be a symlink
+        self._downloaded_data_path = c_path / '_downloaded_data.json'
+
+        # if the local manifest is missing but there are
```

Comment:
As mentioned below, I think we just do away with this warning. The `construct_local_manifest` method gives the user too much opportunity to do something wrong.

Reply:
See above.